test/StringField_test.js

```diff
@@ -211,6 +211,15 @@ describe("StringField", () => {
       Simulate.change(node.querySelector("#root_second"), {target: {value: "0"}});
 
       expect(comp.state.formData).eql("2010-12-01T00:00:00.000Z");
+
+      Simulate.change(node.querySelector("#root_year"), {target: {value: "2012"}});
+      Simulate.change(node.querySelector("#root_month"), {target: {value: "10"}});
+      Simulate.change(node.querySelector("#root_day"), {target: {value: "2"}});
+      Simulate.change(node.querySelector("#root_hour"), {target: {value: "1"}});
+      Simulate.change(node.querySelector("#root_minute"), {target: {value: "2"}});
+      Simulate.change(node.querySelector("#root_second"), {target: {value: "3"}});
+
+      expect(comp.state.formData).eql("2012-10-02T01:02:03.000Z");
```

Comment:
I don't get the purpose of this supplementary test, hint?

Also I usually like to have a single assertion per `it` block, much easier to identify the failing use case.